### DISCUSSION TITLE
Brand guidelines around Llama.cpp logo; is there a media kit that can be downloaded with brand assets?

### DISCUSSION THREAD

**Discussion by jessicachitas** [ggml-org/llama.cpp#10761] [Q&A]
Hi there,

I am working on a website about an AI model that uses Llama.cpp as a runtime and wanted to use the logo in a diagram. 

I also was looking for an svg of this logo with a transparent background in both light and dark variants and could not find anything like that in [this issue](https://github.com/ggerganov/llama.cpp/issues/105).

What are the guidelines regarding the logo and can it be used in this scenario?

Thanks

**Comment by ggerganov** [comment]
We don't have SVGs. Only the PNGs in the [media](https://github.com/ggerganov/llama.cpp/tree/master/media) folder of the repo.

Maybe we can try contacting https://github.com/gojosh for the source material.

**Comment by jessicachitas** [comment]
@gojosh (I hope this might ping him)
If not, I can email him and ask, thank you!

**Comment by jessicachitas** [comment]
Questioned answered! I got the SVGs from Joshua. Thank you!

**Comment by allozaur** [comment]
Small update for anyone concerned  — there's new logo and more branding assets to come :) https://github.com/ggml-org/llama.brand
